content/packages.md

```diff
@@ -93,17 +110,15 @@ There are a handful of other packages that are not in the tidyverse, but are tid
 
 ## Wrangle
 
-As well as [tidyr](http://tidyr.tidyverse.org), and [dplyr](http://dplyr.tidyverse.org), there are five packages designed to work with specific types of data:
+In addition to [tidyr](http://tidyr.tidyverse.org), [dplyr](http://dplyr.tidyverse.org), [stringr](http://stringr.tidyverse.org/), and [forcats](http://forcats.tidyverse.org/), there are three more packages designed to work with specific types of data:
 
-* [stringr](http://stringr.tidyverse.org) for strings.
 * [lubridate](http://lubridate.tidyverse.org) for dates and date-times.
-* [forcats](http://forcats.tidyverse.org) for categorical variables (factors).
 * [hms](https://github.com/tidyverse/hms) for time-of-day values.
 * [blob](https://github.com/tidyverse/blob) for storing blob (binary) data.
 
 ## Program
 
-As well as [purrr](http://purrr.tidyverse.org) which faciliates functional programming, there are three tidyverse packages that help with general programming challenges:
+As well as [purrr](http://purrr.tidyverse.org) which facilitates functional programming, there are three tidyverse packages that help with general programming challenges:
```

Comment:
Is there a pithier description of purrr that doesn't use functional programming? @jennybc do you have one?

Reply:
Not my finest work, but ...

> In addition to purr, which provides very consistent and natural methods for iterating on R objects, there are ...

Reply:
I've made the requested changes. I know we have some related re-wording to do in #77, but is there a reason to hold off on merging this one?

Reply:
I think we should go ahead and merge